footer.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Theme footer template.
+ *
+ * @todo Create proper footer template(s).
+ * @package WPAMPTheme
+ */
+
+// Ignore the issues of this file since that's just a copied HTML placeholder.
```

Comment:
Though this is only a placeholder, it looks good. We could work on dynamic data later, at least after the screencast this Thursday.